dataRetriever/txpool/shardedTxPool_test.go

```diff
@@ -31,35 +31,35 @@ func Test_NewShardedTxPool_WhenBadConfig(t *testing.T) {
 	pool, err := NewShardedTxPool(args)
 	require.Nil(t, pool)
 	require.NotNil(t, err)
-	require.Equal(t, dataRetriever.ErrCacheConfigInvalidSizeInBytes, err)
+	require.Errorf(t, err, dataRetriever.ErrCacheConfigInvalidSizeInBytes.Error())
```

Comment:
...or `require.True(t, errors.Is(err, dataRetriever.ErrCacheConfigInvalidSizeInBytes))`. Can remain like this